### ISSUE TITLE
multiprocessing not working with windows

### ISSUE THREAD

**Issue by papaschlumpf1972** [blue-yonder/tsfresh#981]
**my envirmonment:**
* OS = Windows10, 20H2
* python = 3.9.6
*  IDE = visual studio code
* tsfresh=0.19.0, installed via git

**The problem**:
extract_features() does not work properly unless extract_features(..., n_jobs=0) is used.

if n_jobs > 1, multiprocessing is used. if so the system does not work properly (confusing code execution, extraction is NOT executed).

I did some tests:
* call via command line ==> does not help
* use "__main__" guard ==> does not help
* add freeze_support() ==> does not help

we have a master-thesis and we did a 1st try with tsfresh. unfortunaletly we have large data sets.
with (n_jobs=0) it is working, but it is really sloooooooow.
It would be nice if multiprocessing is supported for windows in a working manner.

**Comment by nils-braun** [comment]
Hi @papaschlumpf1972 ! Sorry for the long delay in answering.

Could you describe what you mean with "confusing code execution"? Do you see any errors? Could you share the code that you are running (preferably a minimal example) with the logs that you are seeing? Thank you!

**Comment by papaschlumpf1972** [comment]
Hi Nils, thx for your answer. "confusing code execution" : I think that the internal stack will be overwritten. I have some some print-statement in the code. When starting tsfresh randomly procedures within the py-code are executed. In the meanwhile I found a solution. I have installed WSL and run my code in the wsl environment. Then everything is running properly ... By the way: I did a lot of tests (VM, Linux, ...). Linux is only ~20% faster comparing to WSL (using all cores-1)...
